etc/system_perf.yml

```diff
@@ -1190,7 +1190,7 @@ buildvariants:
 
   - name: linux-standalone-all-feature-flags
     display_name: Linux Standalone (all feature flags)
-    cron: "0 0 * * *" # Everyday at 00:00
+    cron: "0 0 * * 2,4,6" # Tuesday, Thursday and Saturday at 00:00
```

Comment:
We might actually want this to run everyday? This is presumably where we would want to detect regressions as soon as they land in the code base.

Reply:
[PERF-3328](https://jira.mongodb.org/browse/PERF-3328)

Reply:
We already run 3 replica almost everyday except Sunday. Is your concern the feature flags? With the current process, it takes up to 3 weeks for signal processing and build baron to generate BF tickets for perf issues. I don't think running the test every other day is going to change anything significantly. Actually we hope reducing the total number of change points to triage actually accelerate the whole process.

Reply:
> Is your concern the feature flags?

No, I was thinking that the "all feature flags" variants should be the place where any accidental performance regressions are first caught, resulting in a BF directed to the necessary engineering team. My thinking was similar to what Cris is saying here: https://docs.google.com/document/d/1LEnp9p1yK2t5Hxci1EiJSgbAseNZtlsq0vtE6wraqoE/edit?disco=AAAAgFGat7o

> With the current process, it takes up to 3 weeks for signal processing and build baron to generate BF tickets for perf issues. I don't think running the test every other day is going to change anything significantly.

Oh! I didn't realize that's the current turnaround time we have for getting perf BFs filed and assigned. If that's true, then you're right -- we are probably running the tests too frequently. A few followup thoughts though:

- Is there anything we can be doing to reduce the time between when a regression lands in master and when a BF is directed back to the code author?
- Do the additional test runs help perf build barons at all when they are trying to bisect to find the offending commit?

In any case, I am happy with whatever you decide together with Daly and Cris here.

Reply:
yes, we are working on improving build baron efficiency. reducing the number of change points is a part of it. Stop running unnecessary tests will definitely help. 

Ideally if we can only re-run tests which are potentially introducing a change (we can tell by analyze the latest result), it will definitely accelerate signal processing while not introducing more noise. I was going to give it a try but need to get some other stuffs off my plate first.

Eventually, we will need to stabilize our performance workloads so we can eliminate all "noise filtering" process and get the final results right away.

Reply:
Ack. Resolving the thread.